File: typescript/iniciante/1013.ts
import { bold, red } from "https://deno.land/std@0.224.0/fmt/colors.ts";

const input = (prompt("Digite três números separados por espaço:") ?? "")
  .split(" ")
  .map(Number);

const A = input[0];
const B = input[1];
const C = input[2];

// const largestNumber = Math.max(A, B, C);

let largestNumber = A;

if (B > largestNumber) {
  largestNumber = B;
}

if (C > largestNumber) {
  largestNumber = C;
}

console.log(red(`${bold(String(largestNumber))} eh o maior`));

File: swift/iniciante/1013.swift
import Foundation

if let input = readLine()?.split(separator: " ").map({ Int($0) }) {
    if input.count >= 3, let A = input[0], let B = input[1], let C = input[2] {
        let largestNumber = max(A, B, C)
        print("\(largestNumber) eh o maior")
    }
}
